test/playwright/visual.spec.js

```diff
@@ -48,6 +48,20 @@ test('visual', async ({page, extensionId}) => {
     // Wait for the advanced settings to be visible
     await page.locator('input#advanced-checkbox').evaluate((/** @type {HTMLInputElement} */ element) => element.click());
 
+    // Import jmdict_swedish.zip from a URL
+    await page.locator('.settings-item[data-modal-action="show,dictionaries"]').click();
```

Comment:
is there supposed to be a comma? "show,dictionaries"

Reply:
Yeah it's like that